pkg/publicapi/client/v2/options.go

```diff
@@ -106,11 +145,42 @@ func resolveHTTPClient(config *Options) {
 	config.HTTPClient = defaultHTTPClient(config)
 }
 
+// getTLSTransport builds a http.Transport from the TLS options
+func getTLSTransport(config *Options) *http.Transport {
+	tr := &http.Transport{}
+
+	if !config.TLS.UseTLS {
+		return tr
+	}
+
+	if config.TLS.CACert != "" {
+		caCert, err := os.ReadFile(config.TLS.CACert)
+		if err != nil {
+			panic("invalid ca certificate provided")
```

Comment:
:disappointed: wish we didn't need to panic here, regardless if we keep the panic or not let's be sure to include the error in the message.

Reply:
I've just tried to add the appropriate error returns all the way back up the stack, but it gets more difficult the higher up you go.

Reply:
I ended up removing this by checking the existence of the file denoted by the setting at startup.  It's still there in case the file goes missing in the meantime, but no longer used as an error message mechanism.